styles/Missions/styles.js

```diff
@@ -0,0 +1,16 @@
+import missionsTheme from "../../themes/Missions/theme"
+import ICONS from "../../components/icons/icons"
+
+const missionsStyles = {
+    main:{
+        background:missionsTheme.palette.primary, 
+        height:'2008px', 
+        width:'100%', 
+        alignItems:'center', 
+        textAlign:'center', 
```

Comment:
this should be in the sx prop

`
main: {
....props,
sx: { 
  textAlign: 'center',
  },
`